.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think this can go in a `"prettier": ...` section in `package.json`, if it's not already there.

Reply:
Done in https://github.com/siefkenj/unified-latex/pull/42/commits/574d40963a3a57d91f1018f5dcb6abc0eceb2212